react/e2e/private/notificationsMenu/notificationsMenu.test.js

```diff
@@ -42,7 +42,7 @@ describe('Notifications menu', () => {
     const page2 = await browser.newPage()
     await login(page2, TEST_ACCOUNTS[1], process.env.JEST_SECRET)
 
-    await page2.waitForSelector(SELECTORS.notificationsCounter)
+    await page2.waitFor(3000)
```

Comment:
can wait until [current value != beforeCounter](https://stackoverflow.com/questions/50373158/puppeteer-wait-for-element-to-have-certain-value)